packages/core/test/input.test.ts

```diff
@@ -141,12 +141,16 @@ describe('input', () => {
 
     const observableActor = interpret(observableBehavior, {
       input: { count: 42 }
-    }).start();
+    });
 
-    observableActor.subscribe((state) => {
+    const sub = observableActor.subscribe((state) => {
+      if (state?.count !== 42) return;
```

Comment:
is this one needed?
```suggestion
```

Reply:
Yes, because it is initially called with `undefined`